<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferPage</class>
 <widget class="QWidget" name="TransferPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>529</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="balanceLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>56</y>
     <width>271</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(169, 169, 169);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26px;&quot;&gt;0.00&lt;/span&gt;&lt;span style=&quot; font-size:12px;&quot;&gt; TV&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="addressLabel2">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>134</y>
     <width>65</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(102,102,102);</string>
   </property>
   <property name="text">
    <string>来自地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="balanceLabel2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>64</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(169, 169, 169);</string>
   </property>
   <property name="text">
    <string>账户余额</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="addressLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>500</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(102,102,102);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="changeAccountLabel">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>64</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(102,102,102);</string>
   </property>
   <property name="text">
    <string>当前账户</string>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>309</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(102,102,102);</string>
   </property>
   <property name="text">
    <string>备注</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="messageLineEdit">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>305</y>
     <width>189</width>
     <height>24</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="sendtoLabel">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>176</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(102,102,102);</string>
   </property>
   <property name="text">
    <string>发往</string>
   </property>
  </widget>
  <widget class="QLabel" name="amountLabel">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>219</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(102,102,102);</string>
   </property>
   <property name="text">
    <string>数量</string>
   </property>
  </widget>
  <widget class="QToolButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>380</y>
     <width>94</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{color:white;border:1px solid  rgb(46,78,126);border-radius:6px;
background-color: rgb(46,78,126);}
QToolButton:hover{background-color:rgb(66,98,146);}
QToolButton:disabled{background-color:#cecece;}</string>
   </property>
   <property name="text">
    <string>转账</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="amountLineEdit">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>215</y>
     <width>189</width>
     <height>24</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="feeLabel">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>264</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(102,102,102);</string>
   </property>
   <property name="text">
    <string>手续费</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="feeLineEdit">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>260</y>
     <width>189</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>0.01</string>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel3">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>347</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>余额不足</string>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>174</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>无效地址</string>
   </property>
  </widget>
  <widget class="QToolButton" name="addContactBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>172</y>
     <width>126</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="delegateLabel2">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>23</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>转账</string>
   </property>
  </widget>
  <widget class="QComboBox" name="accountComboBox">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>64</y>
     <width>152</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="sendtoLineEdit">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>172</y>
     <width>189</width>
     <height>24</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>307</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>长度超过40个字节</string>
   </property>
  </widget>
  <widget class="QComboBox" name="assetComboBox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>64</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="assetLabel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>64</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(169, 169, 169);</string>
   </property>
   <property name="text">
    <string>币种</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel6_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>the rate is 0.2%,every transfer accounts minimum fee is 0.1TV</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>amountLineEdit</tabstop>
  <tabstop>feeLineEdit</tabstop>
  <tabstop>messageLineEdit</tabstop>
  <tabstop>sendBtn</tabstop>
  <tabstop>addContactBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
